package.json

```diff
@@ -86,12 +86,12 @@
   "devDependencies": {
     "@appfolio/eslint-config-appfolio-react": "^2.0.2",
     "@babel/cli": "^7.17.6",
-    "@babel/core": "^7.17.8",
-    "@babel/preset-env": "^7.16.11",
+    "@babel/core": "^7.24.7",
```

Comment:
So usually when you update 1 `@babel` package, you should update all of them. So you might also want to update `@babel/cli` and `@babel/preset-react`.

Reply:
👍  done